brain/CMakeLists.txt

```diff
@@ -8,16 +8,22 @@ source_group(\\ FILES CMakeLists.txt)
 
 set(BRAIN_PUBLIC_HEADERS
   circuit.h
-  morphology.h
   types.h
+  cell/morphology.h
+  cell/section.h
+  cell/soma.h
```

Comment:
Except that ellipses require at least 2 more floats to be fully defined (focal distance and angle), which is a waste for millions of points. But let's not discuss this yet.